entities/common.go

```diff
@@ -3,6 +3,6 @@ package entities
 // Cursor represents the pagination cursor on many responses.
 type Cursor struct {
 	Limit  uint64 `json:"limit"`
-	After  uint64 `json:"after"`
-	Before uint64 `json:"before"`
-}
+	After  string `json:"after"`
```

Comment:
This will break backwards comparability. Please restore.

Reply:
So, if I restore it back then I can't deserialize maniphest.search response because it fails to unmarshal it to the uint64 value. 
are you saying we can't change After/Before field types here at all?
If we can't then I'll have to create a separate Cursor type for just maniphest.search methods. 
Please advise.

Reply:
After discussion created new embedded cursor in ManiphestSearchResponse